pangeo_forge_recipes/patterns.py

```diff
@@ -114,8 +135,9 @@ class FilePattern:
       May include ``block_size``, ``username``, ``password``, etc.
     :param query_string_secrets: If provided, these key/value pairs are appended to
       the query string of each ``file_pattern`` url at runtime.
-    :param is_opendap: If True, assume all input fnames represent opendap endpoints.
-      Cannot be used with caching.
```

Comment:
If we wanted to be nice to our users, we would not just remove this but deprecate it. Now that we have a few users, do we want to be more conservative about breaking changes? Or do we just want to move fast and not worry about that.